docs/api/session.md

```diff
@@ -978,7 +978,7 @@ The built in spellchecker does not automatically detect what language a user is
 spell checker to correctly check their words you must call this API with an array of language codes.  You can
 get the list of supported language codes with the `ses.availableSpellCheckerLanguages` property.
 
-**Note:** On macOS the OS spellchecker is used and will detect your language automatically.  This API is a no-op on macOS.
+**Note:** On macOS the OS spellchecker is used and will detect your language automatically.  This API will return whichever languages have been configured by the OS.
```

Comment:
:+1: on explaining that the OS language is used, but this specific phrasing could use revision -- it sounds like it's saying `ses.setSpellCheckerLanguages()` will return whichever languages have been configured by the OS, but it doesn't look like that function returns anything?

Reply:
oops, you're right! i meant to put this in `ses.getSpellCheckerLanguages()`